journal.py

```diff
@@ -30,6 +42,10 @@
 )
 
 
+def add_markdown(text):
+    return markdown.markdown(text, extensions=['codehilite', 'fenced_code'])
+
+
```

Comment:
This really should be a method of the `Entry` class below.  Then you can say this:

```python
entry.markdown()
```

And get the markdown rendered to html from entry.text